### PR TITLE
Fixes a botany runtime

### PR DESCRIPTION
**Pull request by silicons**
fixes a runtime.

### CODE REVIEW

**Comment by ma44** [comment]
WHAT IS THIS

File: code/modules/hydroponics/plant_genes.dm [hunk 1]
```diff
@@ -389,11 +389,14 @@
...
 	var/datum/effect_system/smoke_spread/chem/S = new
 	var/splat_location = get_turf(target)
+	if(!G)
+		return
 	var/smoke_amount = round(sqrt(G.seed.potency * 0.1), 1)
 	S.attach(splat_location)
 	S.set_up(G.reagents, smoke_amount, splat_location, 0)
 	S.start()
-	G.reagents.clear_reagents()
+	if(G)
```
Comment (lzimann) [inline code/modules/hydroponics/plant_genes.dm]: Unless G can get deleted, there's no reason to check again(you already return if G is invalid up there).

Comment (kevinz000) [inline code/modules/hydroponics/plant_genes.dm]: It was saying can't call it on null.reagents.

**Comment by optimumtact** [comment]
This is not a fix, the bug is that Squash is being called when teh host object no longer exists.

You would need to track that down, and it's probably a race condition with qdeleted

**Pull request closed without merging**